tests/data/ptcDet.yaml

```diff
@@ -0,0 +1,136 @@
+AMP:
+  C00:
+    AMP_GAIN: 1.0972
+    AMP_NOISE: 9.1
+    PTC_BFE_A00: -1.5404715564790175e-06
+    PTC_FIT_TYPE: EXPAPPROXIMATION
+    PTC_GAIN: 1.0661058668790242
+    PTC_NOISE: 10.058688412367907
+    PTC_TURNOFF: 123562.9675738056
+  C01:
+    AMP_GAIN: 1.0933
+    AMP_NOISE: 9.9
+    PTC_BFE_A00: -1.2504981154677366e-06
+    PTC_FIT_TYPE: EXPAPPROXIMATION
+    PTC_GAIN: 1.0830855269671449
+    PTC_NOISE: 14.346971655569085
+    PTC_TURNOFF: 113718.47619676767
+  C02:
+    AMP_GAIN: 1.0874
+    AMP_NOISE: 9.4
+    PTC_BFE_A00: -1.215940719362778e-06
+    PTC_FIT_TYPE: EXPAPPROXIMATION
+    PTC_GAIN: 1.0807676290253927
+    PTC_NOISE: 14.03774165657813
+    PTC_TURNOFF: 114201.75619521466
+  C03:
+    AMP_GAIN: 1.0939
+    AMP_NOISE: 9.1
+    PTC_BFE_A00: -1.3035057718913054e-06
+    PTC_FIT_TYPE: EXPAPPROXIMATION
+    PTC_GAIN: 1.084482407768517
+    PTC_NOISE: 13.161173288947262
+    PTC_TURNOFF: 113346.45919560928
+  C04:
+    AMP_GAIN: 1.084
+    AMP_NOISE: 9.7
+    PTC_BFE_A00: -1.3216774105752976e-06
+    PTC_FIT_TYPE: EXPAPPROXIMATION
+    PTC_GAIN: 1.0775677865243305
+    PTC_NOISE: 12.342874592561309
+    PTC_TURNOFF: 118807.93244035645
+  C05:
+    AMP_GAIN: 1.1047
+    AMP_NOISE: 10.3
+    PTC_BFE_A00: -1.2534130929939756e-06
+    PTC_FIT_TYPE: EXPAPPROXIMATION
+    PTC_GAIN: 1.0831545234574225
+    PTC_NOISE: 14.369884603079642
+    PTC_TURNOFF: 119411.37082981659
+  C06:
+    AMP_GAIN: 1.1039
+    AMP_NOISE: 10.0
+    PTC_BFE_A00: -1.1833364688792843e-06
+    PTC_FIT_TYPE: EXPAPPROXIMATION
+    PTC_GAIN: 1.0936583586522444
+    PTC_NOISE: 15.381025016469493
+    PTC_TURNOFF: 118690.37014599473
+  C07:
+    AMP_GAIN: 1.1033
+    AMP_NOISE: 8.5
+    PTC_BFE_A00: .nan
```

Comment:
This is also all nans in the linearity?

Reply:
This I'm not surprised by:  C07 has a bad cameraGeom values, so it generally gets masked.  I can't remember off hand if this is a saturation issue or a read noise issue, but they're both wrong and so this amplifier is bad in ci_cpp processing.  It should be unrelated to the bad linearity fits.